.github/workflows/doc-automation.yml

```diff
@@ -0,0 +1,45 @@
+
+# This action is based on the automation in pytorch/serve. Edit as appropriate
+# TODO: Uncomment this when the automation should run on commits to main
+#on:
+#  push:
+#    branches:
+#      - main
+# TODO: Comment or delete this whole section when automation should run on commits to main
+on: 
+  workflow_dispatch:
+jobs:
+  build_docs_job:
+    runs-on: ubuntu-latest
```

Comment:
Should we lock it in on a specific release to make sure that it wont break in every april/october, when ubuntu  releases new major versions?

Reply:
Same as above